components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -137,6 +179,16 @@ const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment })
           </Button>
         </>
       )}
+      {!canEdit && currentUserId && (
+        <div className="edit-block">
+          <span
```

Comment:
Semantically this `<span>` should be a `<button>`

Reply:
Would you be ok with this being a wee follow-up? I'd like to come through and also convert all the strings to translations, too. I could do that and maybe create a transparent `buttonVariant` that we can use on all these icons? (the `<span>`s are existing from previous work currently)

Reply:
If so then I can create the issue and work on it as one of my next things